scripts/updateRelacionesNacidos.ts

```diff
@@ -0,0 +1,88 @@
+import { Paciente } from '../core-v2/mpi/paciente/paciente.schema';
+import { userScheduler } from '../config.private';
+import { PacienteCtr } from '../core-v2/mpi/paciente/paciente.routes';
+import { ParentescoCtr } from '../core-v2/mpi/parentesco/parentesco.routes';
+
+let dataLog: any = new Object(userScheduler);
+dataLog.body = { _id: null };
+dataLog.method = null;
+
+
+async function run(done) {
+    const progenitor = await ParentescoCtr.findOne({ nombre: '^progenitor' }, {}, userScheduler as any);
+    const hijo = await ParentescoCtr.findOne({ nombre: '^hijo' }, {}, userScheduler as any);
+    const otro = await ParentescoCtr.findOne({ nombre: '^otro' }, {}, userScheduler as any);
+    const cursor = Paciente.find({
+        $and: [
+            { 'relaciones.relacion': null },
+            { 'relaciones.0': { $exists: true } }
+        ]
+    }).cursor({ batchSize: 100 });
+
+    const updateRelacion = async (paciente: any) => {
+        try {
+            let relaciones = paciente.relaciones.map(async (unaRel: any, posRelBebe: any) => {
+
+                let parentezcoRelacion = null;
+                if (!unaRel.relacion && unaRel.referencia) {
+                    // si no tiene una relacion asociada
+                    // creamos la relacion hijo/otro para el familiar relacionado
+                    let pacParentezco = await PacienteCtr.findById(unaRel.referencia);
+
+                    if (pacParentezco) {
+                        const hijoRelacion = {
```

Comment:
Podés declararlo acá como `any`, así te ahorras marcalo as any tantas veces mas abajo.
```suggestion
                        const hijoRelacion: any = {
```

Reply:
cambio realizado